stake-pool/program/src/state.rs

```diff
@@ -115,3 +118,176 @@ impl State {
         }
     }
 }
+
+const MAX_VALIDATOR_STAKE_ACCOUNTS: usize = 1000;
```

Comment:
We might want to have a way to keep this configurable, maybe by setting the expected size from the outside or by having the program figure out how big it should be based on the size of the account passed in

Reply:
Yep, this was my thought as well, decided to keep it simple for now, but might make it configurable for the final version.